dotnet/src/SemanticKernel.Abstractions/AI/TextCompletion/ITextCompletionResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel.AI.TextCompletion;
+
+public interface ITextCompletionResult
+{
+    Task<string> CompleteAsync(CancellationToken cancellationToken = default);
```

Comment:
The method names on these 2 interfaces:
CompleteAsync sounds like this is where you are invoking the work, but I think we're literally just getting a Task or IAsyncEnumerable already in progress, right? If that's the case, then perhaps Get* methods are better here (i.e. GetCompletionAsync(), GetCompletionStreamingAsync) as that sounds more like you're just retrieving the thing instead of invoking a new action on the model.

Reply:
Agreed, renamed them accordingly